### PR TITLE
Add native support for Asp.net core 

### PR DESCRIPTION
**Pull request by Bobreshovr**
#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,45 +45,39 @@ Create ```HorariumServer``` and schedule ```TestJob```
...
 
-    public IDisposable BeginScope()
-    {
-        return _serviceProvider.CreateScope();
-    }
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+//...
+service.AddHorariumServer(MongoRepositoryFactory.Create("mongodb://localhost:27017/horarium"));
```
Comment (itrofimow) [inline README.md]: serviceS

Comment (Bobreshovr) [reply README.md]: done

File: src/Horarium.AspNetCore/RegistrationHorariumExtension.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                settings.JobFactory = new JobFactory(serviceProvider);
+            }
+
+            if (settings.Logger is EmptyLogger)
+            {
+                settings.Logger = new HorariumLogger(serviceProvider.GetService<ILogger<HorariumLogger>>());
+            }
+        }
+    }
+}
```
Comment (mmikirtumov) [inline src/Horarium.AspNetCore/RegistrationHorariumExtension.cs]: new line

File: src/Horarium.Test/AspNetCore/RegistrationHorariumExtensionTest.cs [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                throw new NotImplementedException();
+            }
+
+            public void Error(string message, Exception ex)
+            {
+                throw new NotImplementedException();
+            }
+        }
+    }
+}
```
Comment (mmikirtumov) [inline src/Horarium.Test/AspNetCore/RegistrationHorariumExtensionTest.cs]: new line

**Pull request merged**